config/config.go

```diff
@@ -72,7 +72,7 @@ func SendStats() bool {
 	return globalConf.sendStats
 }
 
-func PrintVersionAndExit() bool  {
+func PrintVersionAndExit() bool {
 	return printVersionAndExit
```

Comment:
warning: exported function PrintVersionAndExit should have comment or be unexported (golint)